sources/api/pluto/src/eks.rs

```diff
@@ -70,3 +69,40 @@ where
         field: "kubernetes_network_config",
     })
 }
+
+#[cfg(not(feature = "fips"))]
+fn build_client<H, N>(
+    https_proxy: Option<H>,
+    no_proxy: Option<&[N]>,
+    config: aws_config::SdkConfig,
+) -> Result<aws_sdk_eks::Client>
```

Comment:
`pluto` needs to arrange to call this before either the EC2 or EKS client is initialized:
```
let _ = rustls::crypto::aws_lc_rs::default_provider().install_default();
```

It's not OK to rely on `imdsclient` to do it; it's essentially a coincidence that all the settings that `pluto` currently deals with have some interaction with IMDS.

Arguably, `imdsclient` shouldn't do it at all and any caller that needs TLS and FIPS should arrange to do it. I'd like to see that change attempted since it's weird to depend on an HTTP-only client to set up something for HTTPS.